Dockerfile

```diff
@@ -56,6 +56,17 @@ ADD ./get /home/ipython/py/get
 ENV PYTHONPATH /home/ipython/py/:$PYTHONPATH
 ENV PATH /home/ipython/py/:$PATH
 
+# ENV variables to replace conf file
+ENV DEBUG false
```

Comment:
In one command, or not at all? https://github.com/bgruening/docker-ipython-notebook/pull/37/files#r29353259